src/components/v2/V2Project/V2ManageTokensSection/V1ProjectTokensSection/MigrateV1ProjectTokensModal/GrantTransferPermissionCallout.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button } from 'antd'
```

Comment:
This component renders this UI

<img width="556" alt="Screen Shot 2022-07-14 at 3 00 14 PM" src="https://user-images.githubusercontent.com/94939382/179124804-93327148-b9bb-41c2-a55d-a7aa25e34793.png">